msrest/exceptions.py

```diff
@@ -27,19 +27,28 @@
 import logging
 import sys
 
+from typing import Callable, Any, Optional, TYPE_CHECKING
```

Comment:
This is not working for Python 3.5.1.

[root@server101 ~]# python3.5
Python 3.5.1 (default, Jul 14 2017, 09:39:18)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-11)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import msrest
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/VRTSpython/lib/python3.5/site-packages/msrest/__init__.py", line 27, in <module>
    from .configuration import Configuration
  File "/opt/VRTSpython/lib/python3.5/site-packages/msrest/configuration.py", line 40, in <module>
    from .exceptions import raise_with_traceback
  File "/opt/VRTSpython/lib/python3.5/site-packages/msrest/exceptions.py", line 30, in <module>
    from typing import Callable, Any, Optional, TYPE_CHECKING
ImportError: cannot import name 'TYPE_CHECKING'

Reply:
Hi, 
You are right about the version problem. 
But we ship python 3.5.1 with our product as a bundle. So we cannot change the version, we may have to ship a patch of there is no other way

Reply:
Hi, 
You are right about the version problem. 
But we ship python 3.5.1 with our product as a bundle. So we cannot change the version, we may have to ship a patch of there is no other way.